Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY --chown=builduser:builduser ./conseil-lorre/src /src/conseil-lorre/src
 COPY --chown=builduser:builduser ./build.sbt /src
 COPY --chown=builduser:builduser ./publishing.sbt /src
 WORKDIR /src
-RUN sbt clean assembly -J-Xss32m
+RUN sbt 'set api / assembly / assemblyOutputPath := file(s"/tmp/conseil-api.jar")' 'set lorre / assembly / assemblyOutputPath := file(s"/tmp/conseil-lorre.jar")' clean assembly -J-Xss32m
```

Comment:
Since this setup depends heavily on `build.sbt` (each change of module name will brake this command for example) and basically `build.sbt` is a "place to go" when it comes to building the project. I would rather suggest to create command alias in the `build.sbt` rather than putting setting properties in here. 

Maybe we could have command: 
- `assembly` which would work as previously, to the `/tmp/` and
- `assemblyDist` which would put the jars into specific `/dist/` folders?

I mean, I'm not sure if it's safe for the community to change the original location, but I'll leave that up to you.

Reply:
I need your opinions on this, I made the change because one of our devs didn't found the assembled jars where expected, somehow....
I thought that putting things in a global tmp dir without any output clearly declaring so could be confusing. My take on that was to move the generated artifacts to a locally distinguished directory, as a default for people or ci doing that locally.

Though, even if the idea of separate tasks is fine, I'm not sure if it's worth it to use /tmp/ as the default choice. In that case I'd simply ignore the PR and leave things as is.

Reply:
I think most misleading part was that `conseil-xyz.jar` was generated in the `target` directory inside of the project. I don't think the output directory of the two other jars is an issue.

Reply:
if that's the case, I can close this issue, with the mental reminder to document where the jars are created in the wiki